rudder-core/src/test/scala/com/normation/rudder/services/policies/write/WriteSystemTechniquesTest.scala

```diff
@@ -97,23 +107,31 @@ import net.liftweb.common.Loggable
 @RunWith(classOf[JUnitRunner])
 class WriteSystemTechniqueSpec extends Specification with Loggable with ContentMatchers with AfterAll {
 
+  //just a little sugar to stop hurting my eyes with new File(blablab, plop)
+  implicit class PathString(root: String) {
+    def /(child: String) = new File(root + "/" + child)
```

Comment:
you are changing the semantic, by replacing File(path, filename) to File(path+"/"+filename)